.template/addons/github/.github/workflows/review_code.yml.tt

```diff
@@ -0,0 +1,57 @@
+name: Review code
+
+on:
+  pull_request:
```

Comment:
Should we change to the following so that this workflow can be run only when we open the PR and update the code?
```suggestion
  pull_request:
    types: [opened, reopened, synchronize]
```

Reply:
how about using the push event instead?
```suggestion
on:
  push
```

Reply:
@longnd I think danger needs a pull request to post the comments, please check it. I remember we had an issue when running danger without PR in `the1-api` project, that why we change to use `pull_request` instead of `push` event

Reply:
yes, that's true :/ Danger check for `pull_request` event 
https://github.com/danger/danger/blob/2d63074b6ccb503642a99aaa40269a84e7067a33/lib/danger/ci_source/github_actions.rb#L23-L25
Fixed in e76bf2f.